app/controllers/users_controller.rb

```diff
@@ -1,2 +1,26 @@
 class UsersController < ApplicationController
+
+	def index
+		@users = User.all
+	end
+
+	def new
+		@user = User.new
+	end
+
+	def create
+		@user = User.find(user_params)
+		redirect_to "/users/#{@user.id}"
+	end
+
+	def show
+		@user = User.find(params[:id])
+	end
+
+	private
+
+	def user_params
+		params.require(:user).permit(:first_name, :last_name, :email, :password_digest, :city_id)
```

Comment:
We aren't going to have a user add a city_id directly. It should be assigned, a bit like the password hash, upon creation. There should probably be a :city field on the sign up form. The :city_id should be removed for now